resources/metadata/apiKeyValueMapDownload.yaml

```diff
@@ -0,0 +1,40 @@
+metadata:
+  name: apiKeyValueMapDownload
+  description: Download a specific Key Value Map from the API Portal
+  longDescription: |
+    With this step you can download a specific Key Value Map from the API Portal, which returns a zip file with the Key Value Map contents in to current workspace using the OData API. Learn more about the SAP API Management API for downloading an Key Value Map artifact [here](https://help.sap.com/viewer/66d066d903c2473f81ec33acfe2ccdb4/Cloud/en-US/e26b3320cd534ae4bc743af8013a8abb.html).
```

Comment:
Please put into multiple lines. The rendered md will put them back into one paragraph, unless there is an empty line between two lines of text.

Reply:
done